Functions/QualityControl/templateConfigReportPDF.json

```diff
@@ -0,0 +1,338 @@
+{
```

Comment:
please note that xASL_io_ReadJson strictly adheres to JSON standard - unlike spm_jsonread.

So don't be surprised if reading some JSONs throws an error.

Reply:
It didn't!